frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                        return;
+                    }
+
+                    var self = this;
+
+                    var points = this.dataPoints.map(function(path, pathIdx) {
+                        var series = path.map(function(val, idx) {
+                            return [idx + 1, val];
+                        });
+                        var pathCopy = _.extend({}, self.paths[pathIdx]);
+                        pathCopy.data = series;
+                        return pathCopy;
+                    });
+
+                    this.unbindResizer();
+
+                    countlyCommon.drawTimeGraph(points,
+                        $(this.$refs.container),
+                        this.bucket, this.overrideBucket,
+                        this.small, null,
+                        this.options);
+
+                    setTimeout(function() {
+                        self.initializeResizer();
+                    }, 0);
+                },
+                initializeResizer: function() {
+                    var plot = $(this.$refs.container).data("plot");
+                    plot.getPlaceholder().resize(this._onResize);
+                },
+                unbindResizer: function() {
+                    var plot = $(this.$refs.container).data("plot");
+                    if (plot) {
+                        plot.getPlaceholder().unbind("resize", this._onResize);
+                    }
+                },
+                _onResize: function() {
+                    var self = this,
+                        plot = $(this.$refs.container).data("plot"),
+                        placeholder = plot.getPlaceholder();
+
+                    if (placeholder.width() === 0 || placeholder.height() === 0) {
+                        return;
+                    }
+
+                    // plot.resize();
+                    // plot.setupGrid();
+                    // plot.draw();
+
+                    var graphWidth = plot.width();
+
+                    $(self.$refs.container).find(".graph-key-event-label").each(function() {
+                        var o = plot.pointOffset({x: $(this).data("points")[0], y: $(this).data("points")[1]});
+
+                        if (o.left <= 15) {
+                            o.left = 15;
+                        }
+
+                        if (o.left >= (graphWidth - 15)) {
+                            o.left = (graphWidth - 15);
+                        }
+
+                        $(this).css({
+                            left: o.left
+                        });
+                    });
+
+                    $(self.$refs.container).find(".graph-note-label").each(function() {
+                        var o = plot.pointOffset({x: $(this).data("points")[0], y: $(this).data("points")[1]});
+
+                        $(this).css({
+                            left: o.left
+                        });
+                    });
+                }
+            },
```

Comment:
Codacy found an issue: [Multiline support is limited to browsers supporting ES5 only.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)